SingularityUI/app/components/common/Link.cjsx

```diff
@@ -0,0 +1,13 @@
+Link = React.createClass
+
+    getLink: ->
+        return window.config.appRoot + "/task/" + @props.taskId
```

Comment:
can just use `config` here, don't need it to be `window.config`

also, another case for the string interpolation here
```coffee
"#{ config.apiRoot }/task/#{ @props.taskId }"
```